doc/source/getting_started/dpf_server.rst

```diff
@@ -120,3 +120,27 @@ DPF Server can be run in a Docker container.
     docker build . -t dpf-core:v2025.1.pre0 --build-arg DPF_VERSION=251
 
 5. To run the DPF Docker container, license it. For more information, see :ref:`DPF Preview License Agreement<target_to_license_terms>`.
+
+.. _ref_dpf_server_secure_mode:
+
+Run DPF Server in Secure mode wih mTLS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting with Ansys 2026 R1 (DPF 2026.1.0, kernel version 11.0) and PyDPF-Core 0.15.0, DPF Server gRPC connections defaults to mTLS Transport.
+
+This change is also brought by service packs for the following older Ansys versions:
+
+- `Ansys 2025 R2 SP04`
```

Comment:
The gRPC changes are also in SP03

Reply:
@rafacanton ah i did not know that.
However, I realized that I am not sure whether I should write that authenticated communication is the new default in the service packs too or if it is just "available" but not the default?

Reply:
the default is in-process as in the previous releases, so no grpc communication. the second is grpc with mtls and the third is the ability to use the grpc unsecure mode